index.js

```diff
@@ -107,6 +107,21 @@ module.exports = function (themeopts) {
       });
       docs.list.push(variables);
     }
+    // force mixins to be last:
+    const max = docs.list.reduce((curMax, section) => {
+      if (!section.order) {
+        return curMax;
+      }
+      if (section.order > curMax) {
+        return section.order;
+      }
+      return curMax;
+    }, 0);
+    docs.list.forEach((section) => {
+      if (section.title === 'Development') {
```

Comment:
can we change this to be more generic.  Maybe pass in a sectionOrder object which takes a title and the order value?